UserInterface/MainWindow.cpp

```diff
@@ -910,6 +919,16 @@ void OpenInfraPlatform::UserInterface::MainWindow::actionGetCameraState() {
 	*/
 }
 
+void OpenInfraPlatform::UserInterface::MainWindow::on_actionDeleteObject_triggered(const std::shared_ptr<oip::IModel>& model, const QString& filename){
+	
+	QMessageBox::information(this, tr("Delete object"),
+		tr("The Button works!"), QMessageBox::Ok);
+	//OpenInfraPlatform::Core::DataManagement::Data::removeModel(model);
+

```

Comment:
I'm quite sure removeModel() will be necessary.
In case it doesn't work you can also try removeallModels(), see if it works and then try to progress from there.

Reply:
auto& data = OpenInfraPlatform::Core::DataManagement::DocumentManager::getInstance().getData();
	data.removeAllModels();

Reply:
fileNames -> Datei löschen?